_overviews/scala3-migration/scaladoc-settings-compatibility.md

```diff
@@ -0,0 +1,55 @@
+num: 24
+previous-page: options-new
+next-page:
+---
+
+The current page is stating status of scaladoc settings. The related Github issue can be found here for [discussion](https://github.com/lampepfl/dotty/issues/11907)
+
+
+| Scala2 | Scala3 | Description | Comment | Is implemented?
+| ------------- | ------------- | --- | --- | --- |
+| -doc-format | _ | Selects in which format documentation is rendered. | Actually old scaladoc supports only html, so it is in some way coherent with new scaladoc, which provides only html | <i class="fa fa-times fa-lg"></i> | 
+| -doc-title | -project | The overall name of the Scaladoc site | Aliased in [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -doc-version | -project-version | | Aliased in [#11965](https://github.com/lampepfl/dotty/issues/11965)  | <i class="fa fa-check fa-lg"></i> | 
+| -doc-footer | _ | A footer on every Scaladoc page, by default the EPFL/Lightbend copyright notice. Can be overridden with a custom footer. | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -doc-no-compile | _ | A directory containing sources which should be parsed for docstrings without compiling (e.g. AnyRef.scala) | We don't need this as we have completely different approach to that issue using -Ydocument-synthetic-types flag for synthetic types | <i class="fa fa-check fa-lg"></i> | 
+| -doc-source-url | -source-links | A URL pattern used to link to the source file, with some variables supported... | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
+| -doc-external-doc | -external-mappings | Comma-separated list of classpath_entry_path#doc_URL pairs describing external dependencies. | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
+| -jdk-api-doc-base | -external-mappings | URL used to link Java API references. | You can specify jdk via -external-mappings since they are generalized setting. | <i class="fa fa-check fa-lg"></i> |
+| -doc-generator | _ | The fully qualified name of a doclet class, which will be used to generate the documentation | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -doc-root-content | -doc-root-content | The file from which the root package documentation should be imported. | |  <i class="fa fa-check fa-lg"></i> |
+| -implicits | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-debug | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-show-all | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-sound-shadowing | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-hide | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
```

Comment:
Maybe explain if there is a replacement available? Also, why wouldn’t we need this in Scala 3?

Reply:
By default we create diagrams

Reply:
Great! Then I suggest to explain it:

```suggestion
| -diagrams | _ |  | Class diagrams are always enabled in Scala 3 | <i class="fa fa-check fa-lg"></i> |
```